schema/whats-new-panel.schema.json

```diff
@@ -8,6 +8,11 @@
       "description": "Plain text (no HTML allowed)",
       "type": "string"
     },
+    "validStringOrID": {
+      "type": "string",
+      "description": "General purpose valid text with verification",
+      "pattern": "^[A-Za-z0-9-_]*$"
```

Comment:
I was thinking we could replace "string" with this definition+pattern check. It's not meant for validation but more for catching mistakes like the "," and so on.

Reply:
This is great, I like it.